src/components/ProfilePage/QuestCard/VoidQuestCard.style.ts

```diff
@@ -8,12 +9,21 @@ export const VoidQuestCardContainer = styled(Box)(({ theme }) => ({
     theme.palette.mode === 'light'
       ? '#FFFFFF'
       : alpha(theme.palette.white.main, 0.08),
-  height: '416px',
-  width: '272px',
+  height: 440,
+  width: 288,
   borderRadius: '24px',
   border: 16,
   borderColor:
     theme.palette.mode === 'light'
       ? theme.palette.white.main
       : alpha(theme.palette.white.main, 0.08),
```

Comment:
alphaLight200. from palette should do the same

Reply:
changed